src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,87 @@
+import {
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import { Loader } from '../../components/Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { PeopleFilters } from '../../components/PeopleFilters';
+import { PeopleTable } from '../../components/PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState(false);
+  const [loading, setLoading] = useState(false);
```

Comment:
Rename all boolean variables according to [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  const [isError, setIsError] = useState(false);
  const [isLoading, setIsLoading] = useState(false);
```